website/app/templates/index.hbs

```diff
@@ -26,7 +26,7 @@
         />
       </div>
     </div>
-    <ul class="doc-page-home__cards" role="list">
+    <ul class="doc-page-home__cards" role="list" aria-label="the cornerstones of the helios design system">
```

Comment:
Just a question: is this all lowercase on purpose?

Reply:
It's lowercase because it doesn't matter in the case of a screen-reader. It would matter if there was an acronym (those should be all uppercase and are read out separately). But in this instance, it is fine. 👍

Reply:
yeah, what I imagined (but wasn't sure). thanks for the clarification